src/servivces/api/todoService.js

```diff
@@ -0,0 +1,42 @@
+import httpClient from '../HttpBaseClient';
+import { UPDATE_TODO } from '../../store/todo/constants';
+
+const ENDPOINTS = {
+  GET_ALL_TODOS: '/todo',
+  UPDATE_TODO: '/todo',
+  DELETE_TODO: '/todo',
+  ADD_TODO: '/todo',
+  AUTH_LOGIN: '/todo',
+  AUTH_LOGOUT: '/todo'
```

Comment:
nema potrebe da naovids rute iste pod razlicitim nazivom, stavi jednu generalnu rutu TODO koja ce da bude bas to, ruta na kojo se pucaju todo request-ovi, ukoliko imas neku metodu koja se razilkuj od ostalih kao npr Delete nju mozes posebnu definisati, isto tako formiraj je do kraja i iskoristi https://millermedeiros.github.io/mdoc/examples/node_api/doc/util.html#util.format